bolt-modules/boltlib/lib/puppet/functions/run_container.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require 'bolt/container_result'
+require 'bolt/error'
+require 'bolt/util'
+
+# Run a container.
+#
+# > **Note:** Not available in apply block
+Puppet::Functions.create_function(:run_container) do
+  # Run a container.
+  # @param image The name of the image to run.
+  # @param options A hash of additional options.
+  # @option options [Boolean] _catch_errors Whether to catch raised errors.
+  # @option options [String] cmd A command to run in the container.
+  # @option options [Hash] env_vars Map of environment variables to set.
```

Comment:
I think it makes sense. `env_vars` _here_ is environment variables to set inside the container. The metaparameter equivalent would be environment variables to set when running the command to start the container (`DOCKER_HOST`, etc). But maybe it should have a different _name_ here?

Reply:
Ya, I'm not opposed to it per se, I just think it's something that could trip users up. A different name might work, but I'm not sure what would make more sense since `env_vars` is about as straightforward as it gets. :P

Reply:
There's a reason I didn't _propose_ a different name :D

Reply:
Documenting it as `env_vars` seems better, and `_env_vars` should also magically work because we strip  :shrug: That does mean that `_env_vars` will override `env_vars` if you have both but like....that's true of any plan options and seems like an unlikely mistake to make.